.osparc/metadata.yml

```diff
@@ -13,19 +13,173 @@ authors:
 inputs:
   input_1:
     displayOrder: 1
-    label: input_1_label
-    description: The input 1 description
-    type: string
-    defaultValue: some_value(optional)
+    label: Input HDF5
+    description: Compatible HDF5 file (check [k-Wave User Manual](http://www.k-wave.org/manual/k-wave_user_manual_1.1.pdf) to know how to create it.)
+    type: data:*/*
     fileToKeyMap:
-      somefilename.ext: input_1
-  
+      input.h5: input_1
+  input_2:
+    displayOrder: 2
+    label: Compression level (-c)
+    description: Compression level
+    type: ref_contentSchema
+    contentSchema: 
+      title: Compression level
+      type: integer
+      default: 1
```

Comment:
@pcrespov, in the UI, I don't see the default value I have here. Would you see why that happens? How can I debug this? I have it running in my local deploy.

Reply:
at first sight it looks correct to me https://json-schema.org/understanding-json-schema/reference/numeric.html

Try this: make `input_2.label` the same as `input_2.contentSchema.title`. Perpahs there is bug ...

Reply:
So it didn't work. I got the impression that we don't do anything with `default` if provided within the `contentSchema`. 

And if understood this well, default is just an "annotation": https://stackoverflow.com/a/72048005

I solved by changing to:
```
  input_2:
    displayOrder: 2
    label: Compression level
    description: Compression level (-c)
    defaultValue: 0
    type: ref_contentSchema
    contentSchema: 
      title: Compression level
      type: integer
      minimum: 0
      maximum: 9
```

Reply:
well I guess this follows the node json schema @pcrespov .